tools/gromacs/energy.xml

```diff
@@ -0,0 +1,117 @@
+<tool id="gmx_energy" name="Extract energy components with GROMACS" version="@VERSION@">
+    <description>from an energy (EDR) file</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <expand macro="requirements" />
+
+    <command detect_errors="exit_code"><![CDATA[
+
+        ln -s '$edr_input' ./edr_input.edr &&
+        echo #echo str($terms).replace(',', ' ')# | gmx energy -f ./edr_input.edr -o ./energy.xvg
+        ## echo #echo ' '.join(str($terms).split(','))# | gmx energy -f ./edr_input.edr -o ./energy.xvg
+
+    ]]></command>
+
+    <inputs>
+        <param argument="edr_input" type="data" format="edr" label="EDR file" help="GROMACS energy binary file"/>
+        <param argument="terms" type='select' label="Terms to calculate" multiple="true" help="Select terms for which values should be calculated">
+            <option value="0">Angle</option>
```

Comment:
is angle a good default choice ... maybe use selected="True" to be more explicit